### PR TITLE
Updated transformer

### PR DESCRIPTION
**Pull request by preinaj**

### CODE REVIEW

File: ADLStream/models/transformer.py [hunk 1]
```diff
@@ -285,6 +285,7 @@ class TransformerModel(tf.keras.Model):
 
     def __init__(
         self,
+        attribute,
```
Comment (pedrolarben) [inline ADLStream/models/transformer.py]: Add 'attribute' to the docstring.

File: ADLStream/models/transformer.py [hunk 2]
```diff
@@ -397,7 +404,9 @@ def train_step(self, data):
         combined_mask = self._create_masks(inp, tar_inp)
         with tf.GradientTape() as tape:
             predictions = self((inp, combined_mask, tar_inp), True)
-
+            print("Shape del inp: ",inp.shape)
+            print("Shape del tar: ",tar.shape)
+            print("Shape del tar_inp: ",tar_inp.shape)
```
Comment (pedrolarben) [inline ADLStream/models/transformer.py]: Please remove these prints

File: ADLStream/models/transformer.py [hunk 3]
```diff
@@ -497,6 +515,9 @@ def Transformer(
     """Transformer
 
     Args:
+        attribute (int): Index of the attribute that we want to predict, if the number of 
+          attributes of the input is different from the ones of the output. 
+          Defaults to None.
         input_shape (tuple): Shape of the input data.
```
Comment (pedrolarben) [inline ADLStream/models/transformer.py]: Sort the arguments of the docstring (Keep the same order as in the function).

File: ADLStream/models/transformer.py [hunk 4]
```diff
@@ -497,6 +515,9 @@ def Transformer(
     """Transformer
 
     Args:
+        attribute (int): Index of the attribute that we want to predict, if the number of
```
Comment (pedrolarben) [inline ADLStream/models/transformer.py]: Could it be a list of integers? (list of indices of the attributes to be used as the target)

**Review by pedrolarben — CHANGES_REQUESTED** [review]
Thanks for the update :)
Check the comments for some minor-changes requests.

**Comment by preinaj** [comment]
I think i added the possibility of using multiples attributes instead of just one, in the case that you have different numbers of attributes in the output and input. I tried it with a couple of examples and i think it's working fine. However, when the experiments are done, I will check with a few more examples to see  if it is really everything alright.

**Comment by pedrolarben** [comment]
Close PR because the last commit has affected to all the files. Please feel free to reopen it.

**Pull request closed without merging**